### ISSUE TITLE
Snip stops generating new track info/gets stuck after latest Windows update

### ISSUE THREAD

**Issue by plagueseason** [dlrudie/Snip#383]
I'm currently having an issue where the track info will eventually stop updating track info when new tracks play. This happens on the 6.10.2 release and the latest 7.0.2 release. It seems to have started after I installed the latest Windows 10 update.

I'm running Windows 10 Pro v 1903 - OS Build 18362.267, and the latest version of Spotify from the Windows app store.

**Comment by ItsMadaleine** [comment]
I'm having a similar issue. The track will update, but only certain tracks... every 10th song or so. Using the latest version of SNIP and Windows 10.

**Comment by plagueseason** [comment]
I'm able to recreate this issue if I close out of Spotify (minimize to system tray). If I minimize Spotify instead, this issue does not occur. Might have something to do with the way it's reading the window title or something.

**Comment by GenesisFR** [comment]
That's normal since @dlrudie removed the code that was using the FindWindow() function from the Win32 API.

Should you want to re-implement it, you can use this code snippet I wrote a while ago: https://gist.github.com/GenesisFR/de0ad5710fb5eb221b5a239e819728bf

**Comment by Raigorn** [comment]
Only updates if it's an existing window, stops when minimized to tray.

**Comment by Raigorn** [comment]
Would it not be preferred to work while in the system tray?

**Comment by plagueseason** [comment]
> 
> 
> Would it not be preferred to work while in the system tray?

Yes, I'd prefer it work either way, but I'm guessing they removed the code that used the FindWindow() function just for the sake of future proofing the app if Spotify decides to change the window title again (which broke Snip for awhile there). If a solution can be found to keep it working while in the systray, that'd be great, I guess more of a feature request/recommendation than a bug at this point though.

**Comment by dlrudie** [comment]
I haven’t necessarily read through all the comments yet.

I know about this issue.

The problem is that when you minimize Spotify to the tray Spotify itself clears the window title of all track info. It never used to do that. It’s some change Spotify made over time.

I can come up with a solution. I just haven’t had free time to sit down lately and mess with it.
